models.py

```diff
@@ -1,7 +1,12 @@
-from sqlalchemy import Column, Integer, String, Float, Boolean
+import enum
+from sqlalchemy import Enum, Column, Integer, String, Float, Boolean
 
 from database import Base
 
+class AccountTyp(str, enum.Enum):
```

Comment:
Just a nitpick: should be AccountTyp**e**?

Reply:
Sorry, i worte it in german:D will change it